cloud_governance/cloud_resource_orchestration/aws/ec2/collect_cro_reports.py

```diff
@@ -252,15 +252,62 @@ def update_in_progress_ticket_cost(self):
             if source_data.get('account_name').lower() in self.__account_name.lower():
                 ticket_opened_date = datetime.strptime(source_data.get('ticket_opened_date'), "%Y-%m-%d")
                 duration = int(source_data.get('duration', 0))
-                user_project = source_data.get('project')
                 group_by_tag_name = self.COST_EXPLORER_TAGS[self.TICKET_ID_KEY]
                 user_cost = self.get_user_cost_data(group_by_tag_name=group_by_tag_name, group_by_tag_value=ticket_id, requested_date=ticket_opened_date)
-                user_forecast = self.get_user_cost_data(group_by_tag_name=group_by_tag_name, group_by_tag_value=ticket_id, requested_date=datetime.utcnow(), forecast=True, duration=duration)
-                update_data = {'actual_cost': user_cost, 'forecast': user_forecast, 'timestamp': datetime.utcnow(), f'TotalCurrentUsage-{datetime.utcnow().year}': total_account_cost}
+                user_daily_cost = eval(source_data.get('user_daily_cost', "{}"))
+                user_name = source_data.get('user')
+                ce_user_daily_report = self.__get_user_daily_usage_report(days=4, group_by_tag_value=ticket_id,
+                                                                          group_by_tag_name=group_by_tag_name,
```

Comment:
Why you are fetching only last 4 days, don't we need to fetch last 30 days ?

Reply:
No, 30 days cost is too heavy to fetch. I thought last 4 days is good enough

Reply:
So Why we need to fetch last 4 days, is last 1 day not enough ?

Reply:
we know that, aws cost having the delay of 2 days.

Reply:
So lets choose 3 days